File: Domains/Algorithms/Warmup/TimeConversion.cs
using System;
using System.Collections.Generic;
using System.IO;
class Solution 
{
    static void Main(String[] args) 
    {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
        var hourAndTimeStr = Console.ReadLine();
        var splitHour = hourAndTimeStr.Split(':');
        var hour = Convert.ToInt32(splitHour[0]);
        var time = splitHour[splitHour.Length - 1].Substring(splitHour[splitHour.Length - 1].Length - 2);
        if(time.Equals("PM") && hour != 12)
            splitHour[0] = (hour + 12).ToString();
        else if(time.Equals("AM") && hour == 12)
                splitHour[0] = "00";
            
        Console.WriteLine(string.Format("{0}:{1}:{2}", splitHour[0], splitHour[1], splitHour[splitHour.Length - 1].Substring(0, 2)));
        
    }
}

File: Domains/Algorithms/Warmup/TimeConversion.fs
//Enter your code here. Read input from STDIN. Print output to STDOUT
open System

[<EntryPoint>]
let main args =
    let timeArray = Console.ReadLine().Split(':')
    let timeDay = timeArray.[2].Substring(2)
    let hour = timeArray.[0] |> int
    if hour = 12 && timeDay.Equals("AM") then
        printfn "%s:%s:%s" "00" timeArray.[1] (timeArray.[2].Substring(0,2))
    else if hour <> 12 && timeDay.Equals("PM") then
        printfn "%d:%s:%s" (hour + 12) timeArray.[1] (timeArray.[2].Substring(0,2))  
    else
          printfn "%s:%s:%s" timeArray.[0] timeArray.[1] (timeArray.[2].Substring(0,2))
    0
